Header header
uint16 reg_pwr_total_current
uint16 reg_motor_fb_rpm_left
uint16 reg_motor_fb_rpm_right
uint16 reg_flipper_fb_position_pot1
uint16 reg_flipper_fb_position_pot2
uint16 reg_motor_fb_current_left
uint16 reg_motor_fb_current_right
uint16 reg_motor_charger_state
uint16 reg_power_a_current
uint16 reg_power_b_current
uint16 reg_motor_flipper_angle
int16 battery_current_a
int16 battery_current_b